k8s-scheduler/src/main/resources/scheduler_tables.sql

```diff
@@ -225,6 +225,26 @@ select
 from pod_info
 where status = 'Pending' and node_name is null and schedulerName = 'dcm-scheduler';
 
+create view assigned_pods as
+select
+  pod_name,
+  status,
+  node_name,
+  namespace,
+  cpu_request,
+  memory_request,
+  ephemeral_storage_request,
+  pods_request,
+  owner_name,
+  creation_timestamp,
+  has_node_selector_labels,
+  has_pod_affinity_requirements,
+  has_pod_anti_affinity_requirements,
+  equivalence_class,
+  qos_class
+from pod_info
+where node_name is not null and schedulerName = 'dcm-scheduler';
```

Comment:
schedulerName = 'dcm-scheduler' shouldn't be in the predicate. We want to check against all running pods.